build-ami.sh

```diff
@@ -1,15 +1,22 @@
 #!/bin/bash
-set -ex
+set -exu
 
-APPLIANCE_IMAGE=${1-registry.tld/conjur-appliance}
-APPLIANCE_IMAGE_TAG=${2-latest}
+IMAGE="$1"  # ex: registry.tld/conjur-appliance:4.9-stable
+TAG="${IMAGE##*:}"
 
 if [ ! -f conjur-appliance.tar.gz ]; then
-  docker pull ${APPLIANCE_IMAGE}:${APPLIANCE_IMAGE_TAG}
-  docker save ${APPLIANCE_IMAGE}:${APPLIANCE_IMAGE_TAG} > conjur-appliance.tar
+  docker pull $IMAGE
+  docker save $IMAGE > conjur-appliance.tar
```

Comment:
we can deal with it being a bit slow i think. i'd rather merge this and tackle speeding this up in another issue. jason, could you create that?